### PR TITLE
Update `d2l-rubric` to `4.x`

### PR DESCRIPTION
**Pull request by devpow112**
This is the lit version of rubrics with the LD flag for switching back to the polymer version if needed.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +79,8 @@
     "d2l-program-outcomes-picker": "Brightspace/program-outcomes-picker#semver:^3",
     "d2l-quiz-components": "Brightspace/quiz-components#semver:^1",
     "d2l-resize-aware": "BrightspaceUI/resize-aware.git#semver:^1",
-    "d2l-rubric": "Brightspace/d2l-rubric#semver:^3",
+    "d2l-rubric": "github:Brightspace/d2l-rubric#semver:^4",
+    "d2l-rubric-polymer": "github:Brightspace/d2l-rubric#semver:^3",
```
Comment (devpow112) [inline package.json:83]: This is needed for now since we are feature flagging stuff.

File: package.json [hunk 2]
```diff
@@ -59,6 +58,7 @@
     "d2l-copy-role": "Brightspace/copy-role#semver:^1",
     "d2l-cpd": "Brightspace/continuous-professional-development#semver:^3",
     "d2l-custom-early-progress-report": "Brightspace/custom-early-progress-report#semver:^1",
+    "d2l-custom-my-courses-in-other-orgs": "github:Brightspace/custom-my-courses-in-other-orgs.git#semver:^1",
```
Comment (devpow112) [inline package.json:61]: Just some npm sorting when running npm install/update

Comment (imcdonald-d2l) [reply package.json:61]: Looks like `github:` was appended with the sort as well. Is that intented?

Comment (devpow112) [reply package.json:61]: Yeah it was done automatically by npm. It seems to want `github:` as a prefix. It helps it determine which git endpoint to use.

**Review by imcdonald-d2l — APPROVED** [review]

**Pull request merged**